library/mageia

```diff
@@ -1,4 +1,6 @@
-# maintainer: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
+Maintainers: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
+GitRepo: https://github.com/juanluisbaptiste/docker-brew-mageia.git
 
-latest: git://github.com/juanluisbaptiste/docker-brew-mageia@85a4253a175cde455b9926b4a83fb8d5ceea096f 5
-5: git://github.com/juanluisbaptiste/docker-brew-mageia@85a4253a175cde455b9926b4a83fb8d5ceea096f 5
+Tags: 5,5.1, latest
+GitCommit: 12caf265c2a969c8e50043eae84aa3fbbcdc40e6
```

Comment:
I think you meant `c01771b53f87dd8d130879122d13025a9d957000` from the dist branch.  Which means you'll probably also need a `GitFetch: refs/heads/dist` line.